spec/acts_as_spec.rb

```diff
@@ -4,7 +4,9 @@
   subject { Pen }
 
   let(:pen_attributes) { {name: 'pen', price: 0.8, color: 'red'} }
+  let(:eraser_attributes) { {name: 'eraser', price: 1.2, strength: 3} }
```

Comment:
feat(test): add new model to demo the need of actable_type
can squash with
feat(test): add and modify tests

They are all under the same context.